src/requirements/data/constants.ts

```diff
@@ -7,3 +7,28 @@ export const CREDITS_COURSE_ID = 10;
  * Course id for special course that satisfies FWS requirement.
  */
 export const FWS_COURSE_ID = 11;
+
+/** The real course ID of the course that fulfills swim test. */
+export const SWIM_TEST_COURSE_ID = 350002;
+
+export const ALL_ROSTERS: readonly string[] = [
+  'FA14',
+  'SP15',
+  'SU15',
+  'FA15',
+  'SP16',
+  'SU16',
+  'FA16',
+  'SP17',
+  'SU17',
+  'FA17',
+  'SP18',
+  'SU18',
+  'FA18',
+  'SP19',
+  'SU19',
+  'FA19',
+  'SP20',
+  'FA20',
+  'SP21',
+];
```

Comment:
Were these roster values hardcoded? Do you think it would be good to generate this list of rosters based on the rosters we get from the fetcher (i.e. the filtered-course files)?